tests/sentry/api/test_event_search.py

```diff
@@ -2589,6 +2604,20 @@ def test_average_range(self):
             ]
         ]
 
+        # Test a non duration field
```

Comment:
might be a little redundant, but should we add the analogous test for all the `*_range` functions?